pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -v -rf --ff --cov-report=term-missing
+testpaths =
+    tests/unit_tests*
+norecursedirs = xarray_utils
```

Comment:
Why is this a "norecursedir" ?

Reply:
I don't think this has any effect in this repo. Maybe it got carried over from another repo? If so, which and is it actually needed there?

I think we could delete this if we want.

Reply:
It is geoips repo. @AJThorpe  do you know why xarray_utils has norecursedirs on it ?

Reply:
It had only basic functionality for the xarray to datatree framework, we never developed the unit testing for it since it was a test feature.

Reply:
Future note: It would make sense to develop unit tests for the data.py  and time.py section since there would be a lot of test cases for sectoring and utilities.

Reply:
I think this is fine for now.